other/js/auth_plugin_examples.js

```diff
@@ -0,0 +1,90 @@
+/*
+ * An auth plugin must be a function which returns a function conforing to the
```

Comment:
Fixed in https://github.com/novnc/websockify/pull/285/commits/dc31fab663e7883e5a72ad15f044ffbf4a4e4b18